apps/lfx-one/src/app/modules/meetings/meeting-join/meeting-join.component.html

```
@@ -767,6 +719,56 @@ <h3 class="text-gray-700 text-sm font-medium flex-1">Meeting Materials</h3>
+      <ng-template #header>
+        <div class="flex items-center justify-between w-full">
+          <div class="flex items-center gap-3">
+            <div class="bg-blue-600 text-white rounded-full w-12 h-12 flex items-center justify-center">
+              <i class="fa-light fa-user-group text-white text-2xl"></i>
+            </div>
+            @if (isPastMeeting()) {
+              <div class="flex flex-col">
+                <h1>Meeting Participants</h1>
+                <span class="text-base text-gray-500">{{ attendedCount() }} attended / {{ pastMeetingInvitedCount() }} invited</span>
+              </div>
+            } @else if (meeting().committees && meeting().committees.length > 0) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)